server/plugin/util.go

```diff
@@ -193,3 +194,75 @@ func filterSubscriptionsOnRecordData(subscripitons []*serializer.SubscriptionRes
 
 	return subscripitons[:n]
 }
+
+func (p *Plugin) handleClientError(err error, w http.ResponseWriter, r *http.Request, userID string, isSysAdmin bool, statusCode int, response string) string {
```

Comment:
I think its better to put 2 string arguments together and put the error at end
```suggestion
func (p *Plugin) handleClientError(w http.ResponseWriter, r *http.Request, userID, response string, isSysAdmin bool, statusCode int, err error) string {  
```